xbmc/settings/dialogs/GUIDialogContentSettings.h

```diff
@@ -76,6 +74,10 @@ class CGUIDialogContentSettings : public CGUIDialogSettingsManualBase
   void InitializeSettings() override;
 
 private:
+  void setLabel2(const std::string &settingid, const std::string &label);
+  void toggleState(const std::string &settingid, bool enabled);
+  void setFocus(const std::string &settingid);
```

Comment:
fixed casing in https://github.com/xbmc/xbmc/commit/d56475614c958b2f5334161ea66993018a6884e5